e2e/cypress/integration/channel_settings/channel_header_spec.js

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {getRandomId} from '../../utils';
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+// Group: @channel_settings
+describe('Channel Settings', () => {
+    let testTeam;
+    let user1;
+    let admin;
+
+    before(() => {
+        cy.apiGetMe().then(({user: adminUser}) => {
+            admin = adminUser;
+
+            cy.apiInitSetup().then(({team, user}) => {
+                testTeam = team;
+                user1 = user;
+
+                cy.visit(`/${testTeam.name}/channels/town-square`);
+            });
+        });
+    });
+
+    beforeEach(() => {
+        cy.apiAdminLogin();
+    });
+
+    it('MM-T1808 Hover effect exists to add a channel description / header (when not already present)', () => {
+        // # Create a new public channel and then private channel
+        ['O', 'P'].forEach((channelType) => {
+            cy.apiCreateChannel(testTeam.id, `chan${getRandomId()}`, 'chan', channelType).then(({channel}) => {
+                // # Go to new channel
+                cy.visit(`/${testTeam.name}/channels/${channel.name}`);
+
+                // # Do the header test with public and private channel respectively
```

Comment:
```suggestion
                // * Test hovering over the header with public and private channel respectively
```
can you fix other instances as well, we should be clear on what we are testing so people can easily understand what's going on.

Reply:
✅  Updated as per request/discussion